src/core/utils/text-style.ts

```diff
@@ -16,13 +18,13 @@ export const DEFAULT_TEXT_STYLE: TextStyle = {
     strokeThickness: 0
 };
 
-export function getTextStyle(style: TextStyle = DEFAULT_TEXT_STYLE) {
+export function getTextStyle(style: TextStyle = DEFAULT_TEXT_STYLE): Partial<PIXI.ITextStyle> {
+    const fontWeight = style.fontWeight ?? DEFAULT_TEXT_STYLE.fontWeight;
     return {
         fontSize: `${style.size ?? DEFAULT_TEXT_STYLE.size}px`,
         fontFamily: style.fontFamily ?? DEFAULT_TEXT_STYLE.fontFamily,
-        fontWeight: style.fontWeight ?? DEFAULT_TEXT_STYLE.fontWeight,
+        fontWeight: fontWeight === 'light' ? 'lighter' : fontWeight,
```

Comment:
This was a bug apparently, or it's a bug with the type definitions for `PIXI.ITextStyle` which doesn't include `light` and only `lighter`.

Reply:
Ah, thanks for the correction. I can confirm the 'light' is not listed in the PIXI documentation (at least in the current version).

Reply:
Are you ok with this approach? This way we keep our API consistent but adapt it sufficiently for PIXI.

Reply:
Yes, I am okay with it.

I am thinking that we can later better unify the `TextStyle` with the `PIXI.ITextStyle`.